src/ralph/static/src/scss/_overrides.scss

```diff
@@ -77,3 +77,15 @@ select {
         float: left;
     }
 }
+.result-list {
+    overflow-y: auto;
+}
+.bulk-edit-fill {
```

Comment:
move to ``./components/bulk_edit.scss`` add some namespace (e.g. ``.bulk-edit``) and include in ``ralph.scss``